tools/gcc-bridge/compiler/src/main/java/org/renjin/gcc/symbols/UnitSymbolTable.java

```diff
@@ -25,7 +33,7 @@
   private final GlobalSymbolTable globalSymbolTable;
   private String className;
   private final Map<Integer, Expr> variableMap = Maps.newHashMap();
-  private final Map<String, FunctionGenerator> functions = Maps.newHashMap();
+  private final Map<String, List<FunctionGenerator>> functions = Maps.newHashMap();
```

Comment:
Ok, so GCC is actually emitting overloaded functions with the unmangled names? I was expecting to get function unique function names with argument types encoded? Like: 
https://en.wikipedia.org/wiki/Name_mangling#Simple_example

But this is probably better for us in that the Java will look a bit more like C++ instead of having methods with weird names like __f_v() or __f_i().